site/en/guides/index.md

```diff
@@ -0,0 +1,218 @@
+# Get Started
+
+## Learn about Thread
+
+Are you new to Thread<sup>®</sup>? Or simply need to refresh your knowledge?
+Check out our [Thread Primer](thread-primer/index.md), which covers all the
+basics of Thread and how it works.
+
+## Try OpenThread
+
+Want to see what OpenThread released by Google is all about? The quickest way to
+do so is to run through one of our Codelabs or Guides.
+
+### Simulation Codelab with Docker
+
+Try OpenThread without the need for test hardware. Using Docker on a Mac or
+Linux machine, learn how to:
+
+*   Simulate a Thread network
+*   Authenticate Thread nodes with Commissioning
+*   Use OpenThread Daemon to manage a simulated Thread network featuring an RCP
+
+<a class="button button-primary"
+   href="../codelabs/openthread-simulation/index.lab.md">Try the Simulation Codelab with
```

Comment:
Thinking about this, I've changed my mind.  Let's point any links to Codelabs to the version on openthread.io rather than the version on GitHub.  Mainly because if we want someone to go through a Codelab, we'd want them to use the nicer-looking version.  Please update all the Codelab links here to be the full openthread.io links.

For non-Codelab content we'll point to GitHub if available as normal.

Reply:
OK makes sense; will do...